pallets/nft-sales/Cargo.toml

```diff
@@ -18,8 +18,16 @@ frame-support = { git = "https://github.com/paritytech/substrate", default-featu
 frame-system = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.16" }
 sp-runtime = { git = "https://github.com/paritytech/substrate", default-features = false , branch = "polkadot-v0.9.16" }
 
+# Orml crates
+orml-tokens = { git = "https://github.com/open-web3-stack/open-runtime-module-library", default-features = false, branch = "polkadot-v0.9.16" }
+orml-traits = { git = "https://github.com/open-web3-stack/open-runtime-module-library", default-features = false, branch = "polkadot-v0.9.16" }
```

Comment:
These should probably be optional and included only when benchmarking is enabled?

Reply:
So I'm pretty sure I tried to set these or another pallet only used for benchmarking as optional but that didn't work. Maybe I forgot something else?

Reply:
Got it 🕺  thanks!